### PR TITLE
add matterbridge info

### PR DESCRIPTION
**Pull request by gi-yt**

### CODE REVIEW

File: servers/apps/matterbridge/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Instructions for bridging if topic as group is used:
+
+Prerequisites:
+
+- Access to falcon server of FOSS United
```
Comment (rahulporuri) [inline servers/apps/matterbridge/README.md:5]: not sure where the falcon server is - is this on linode?

File: servers/apps/matterbridge/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- Access to the @fu-bot:tchncs.de account on matrix
```
Comment (rahulporuri) [inline servers/apps/matterbridge/README.md:6]: I don't know who all have access to these credentials - I definitely don't have them and they're not stored on the FOSS United vault from what I can see.

File: servers/apps/matterbridge/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- Admin access in the City Chapters Space
```
Comment (rahulporuri) [inline servers/apps/matterbridge/README.md:7]:
```suggestion
- Admin access in the Matrix Room or Space
```

the advice isn't specific to City Chapters so we can make the language generic

File: servers/apps/matterbridge/matterbridge.toml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+HTMLDisable=false
+# UseUserName shows the username instead of the server nickname
+UseUserName=false
+KeepQuotedReply=true
+ShowJoinPart=true
+
+[telegram]
+[telegram.main]
+Token="REDACTED"
```
Comment (rahulporuri) [inline servers/apps/matterbridge/matterbridge.toml:15]: is this token associated with an individual user or is there a bot account like the fu-bot matrix account?

**Review by rahulporuri — APPROVED** [review]
Thank you very much for writing this up @gi-yt ! We will hopefully be able to add more information once someone else tries to replicate the process!

File: servers/apps/matterbridge/README.md (hunk 1 above)
Comment (gi-yt) [reply servers/apps/matterbridge/README.md:5]: yes. 172.105.55.120. It hosts Vaultwarden and wisharya.com as well

File: servers/apps/matterbridge/README.md (hunk 2 above)
Comment (gi-yt) [reply servers/apps/matterbridge/README.md:6]: I think I gave access to ansh and mangesh, not sure who/where I have to give the credentials to exactly

File: servers/apps/matterbridge/matterbridge.toml (hunk 4 above)
Comment (gi-yt) [reply servers/apps/matterbridge/matterbridge.toml:15]: This is a bot account generated by telegram's BotFather and is tied to a specific telegram user. Its currently linked to my telegram account to be specific.

**Comment by gi-yt** [comment]
@rahulporuri any update on this?

**Pull request merged**